atomix/cluster/src/main/java/io/atomix/cluster/protocol/SwimMembershipProtocol.java

```diff
@@ -69,6 +70,12 @@
 
   public static final Type TYPE = new Type();
   private static final Logger LOGGER = LoggerFactory.getLogger(SwimMembershipProtocol.class);
```

Comment:
:wrench: To ensure we can always configure the loggers as a block, I would also rename this one. Either just `io.atomix.cluster.protocol.swim`, or `io.atomix.cluster.protocol.swim.protocol`.

```suggestion
  private static final Logger LOGGER = LoggerFactory.getLogger("io.atomix.cluster.protocol.swim.protocol");
```